### DISCUSSION TITLE
Supabase was deprecated on Vercel

### DISCUSSION THREAD

**Discussion by scallopy** [supabase/supabase#20968] [General]
Supabase Integration was deprecated on Vercel. After that all our deployments crached.

**Comment by GaryAustin1** [comment]
Can you point to where it has been deprecated?

**Comment by scallopy** [comment]
Hi, Gary
Sure: in my project:
https://vercel.com/rate-me-team/rateme-project-server/settings/integrations
When I try to install it again I cannot do that.
Alson when I browse in Vercel market integrations there is no ability to
integrate Supabase: https://vercel.com/integrations
I tried to install supabase again from here:
https://vercel.com/integrations/supabase , but it was not successful.

Please ask any clarifying questions if needed.
Kind Regards
Petya Velikova

**Reply by GaryAustin1** [reply]
So I don't see anything about it being deprecated.
Are you maybe hitting this issue?
![image](https://github.com/supabase/supabase/assets/54564956/0d44e26f-9fdf-4869-a0c4-f385d41fa255)
https://status.supabase.com/incidents/sgv32x4cpgd8

**Reply by GaryAustin1** [reply]
Also I ran the intergration (I think as I don't use any of this) with an existing SB project and it seemed to link up: 
![image](https://github.com/supabase/supabase/assets/54564956/56a9bee3-850e-4f08-8a14-4c733abcf0b9)

But I have no idea what that means in relation to what you are doing.
Anyway you did not have this as a help question or seem to ask for help in your first post.  I was just trying to understand if something was announced as deprecated.  I'm not qualified to help with Vercel issues.

**Comment by scallopy** [comment]
Only here I see that Supabase is deprecated: See attached image: this is in
our project/settings/integrations or in our case:
https://vercel.com/rate-me-team/rateme-project-server/settings/integrations
Also our deployments start crashing from Jan 17 2024. We investigated a lot
of things there in our project setups on Vercel. The supabase is the last
thing that I investigate now.
My mistake was that when I received this from supabase on Jan 11: Update
your Supabase Connection String - Deadline Extended  I updated
Environment Variables following the link
https://supabase.com/dashboard/project/_/settings/database  maybe 6 days
ago and three days later I found that django admin cached, when I set on
localhost variables to their old version, django admin start working as
expected. But because we need to have a new deployment on Vercel to change
Environment Variables again I can not reload them.

**Comment by scallopy** [comment]
Hi Gary,
Thank you for that example above. I run integrations from Vercel. But I
didn't see this view in the image you sent me there. I will discuss this
with our team and get back to you maybe on Monday.

Kind Regards
Petya Velikova

**Reply by GaryAustin1** [reply]
You might want to start a new question instead of general comment then at that time.  As I said I don't know much about Vercel so getting back with me is a waste of your time on this particular issue.

**Comment by scallopy** [comment]
Ok.It was nice to meet you here. You help me a lot. Thank you very much.
